siriuspy/siriuspy/dvfimgproc/meas.py

```diff
@@ -242,6 +246,13 @@ def process_image(self, **kwargs):
         if self._callback:
             self._callback()
 
+    def _start_imgproc(self, *args, **kwargs):
+        """."""
+        if self._thread and self._thread.is_alive():
+            return
+        self._thread = _Thread(target=self.process_image, daemon=False)
```

Comment:
why do we need Daemon=False here?

Reply:
I told ximenes to put false, but I meant true... :D

Reply:
@xresende, could you change to `daemon=True` in the code, please?